metaflow/plugins/kubernetes/kubernetes.py

```diff
@@ -167,6 +168,7 @@ def create_job(
         persistent_volume_claims=None,
         tolerations=None,
         labels=None,
+        annotations=None,
```

Comment:
Not an issue; I think this is unnecessary, as the job will never receive annotations passed into it? The only source for custom annotations being from the environment variable

Reply:
remove this for annotations as well as for labels since they are both unused